package.json

```diff
@@ -1,7 +1,8 @@
 {
-  "name": "smile-dashboard",
+  "name": "smile-dashboard-client",
   "version": "0.1.0",
   "private": true,
+  "proxy": "http://localhost:4000",
```

Comment:
This will allow us to make requests to our Node server without having to provide the origin it is running on (http://localhost:4000) every time we make a network request to it: